sdk/storage_internal.go

```diff
@@ -338,3 +338,40 @@ func (ref *downloadInfoDTOs) toStruct(networkType NetworkType) ([]*DownloadInfo,
 
 	return drives, nil
 }
+
+type drivesPageDTO struct {
+	Drives []driveDTO `json:"data"`
+
+	Pagination struct {
+		TotalEntries uint64 `json:"totalEntries"`
+		PageNumber   uint64 `json:"pageNumber"`
+		PageSize     uint64 `json:"pageSize"`
+		TotalPages   uint64 `json:"totalPages"`
+	} `json:"pagination"`
+}
+
+func (t *drivesPageDTO) toStruct(networkType NetworkType) (*DrivesPage, error) {
+	page := &DrivesPage{
+		Drives: make([]Drive, len(t.Drives)),
```

Comment:
Better to have []*Drive

Reply:
I do not hink so, because we are missing pagination details in this way =))

Reply:
Which details do you mean?

Reply:
`TotalEntries uint64
	PageNumber   uint64
	PageSize     uint64
	TotalPages   uint64`

Reply:
I meant []Drive -> []*Drive